accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -601,6 +605,28 @@ If _Dm_ is a method with function type `F`, and `args` exists, the static
 analysis of the extension type member invocation is the same as that of an
 invocation with argument part `args` of a function with the given type.
 
+### Special-case Typing Rules for `num` Members
+
+The language specification contains special typing rules for invocations
+of `+`, `-`, `*`, `%`, `remainder` and `clamp` methods on 
+subtypes of `num`, `int` and `double`.
+
+Those rules do *not* apply to invocations of extension or extension type
+members.
+The rules are otherwise unchanged when applied to non-extension type
+member invocations, which will necessarily invoke the corresponding members
+of an instance of `int` or `double`.
+
+_It's not currently possible to have a subtype of `num` which has
+an extension member for those member names, but if it ever becomes possible,
+the rules will still not apply to such an invocation._
+
+_It's currently also not possible to introduce a subtype of `num`, `int` or
+`double`, which does not have the same member signatures for the special-cased
+members as the type that it subtypes, so the special-case rules are safe to
+assume that signature. If we introduce the ability to narrow or change the
+signatures of non-extension-type members of extension types, 
+the special-case typing rules will have to be re-visited._
```

Comment:
Perhaps:
```suggestion
the special-case typing rules may have to be re-visited._
```

We could claim that this case has already been covered: An extension type subtype of `int`, say, can certainly declare an extension type operator `+` with a different signature and implement it.

It would be a new thing if we allow, for example, an _abstract_ declaration with the given name in an extension type declaration to modify the static analysis of a member whose implementation is provided by a non-extension superinterface. However, I'd suggest that we treat this case as an implicitly induced forwarding method; for instance, it would be confusing if such a member is torn off and the resulting function object doesn't have the function type of the member as declared.